pkgs/tools/misc/passmark-performancetest/default.nix

```diff
@@ -0,0 +1,57 @@
+{ lib
+, stdenv
+, fetchurl
+, curl
+, unzip
+, ncurses5
+, autoPatchelfHook
+}:
+let
+  version = "10.2.1003";
+  versionUri = builtins.replaceStrings ["."] ["_"] version;
```

Comment:
The latest version is 11.0.1002 but sadly it only appears to be available via unversioned links :/

https://www.passmark.com/downloads/pt_linux_x64.zip

Reply:
Ah they changed their versioning recently.

Reply:
After some discussion, we would keep the current version as is since unversioned links is hard to track

Reply:
nope, that is not the case. Just upload them to web.archive.org but we loose auto updates.

Reply:
Done